blocks/library/table/index.js

```diff
@@ -69,22 +90,40 @@ export const settings = {
 						onChange={ updateAlignment }
 					/>
 				</BlockControls>
+				<InspectorControls>
+					<PanelBody title={ __( 'Table Settings' ) } className="blocks-table-settings">
```

Comment:
According to the coding styles we follow this `className` should be `blocks-table__settings`, right?

Reply:
Good call.